decidim-proposals/app/helpers/decidim/proposals/proposals_helper.rb

```diff
@@ -17,6 +17,28 @@ def proposal_has_costs?
           translated_attribute(@proposal.execution_period).present?
       end
 
+      def toggle_view_mode_link(current_mode, target_mode, title)
+        path = proposals_path(view_mode: target_mode)
+        icon_name = target_mode == "grid" ? "layout-grid-fill" : "list-check"
+        icon_class = "view-icon--disabled" unless current_mode == target_mode
+
+        link_to path, title: do
```

Comment:
Let's make this AJAX so its consistent with the "order by" UX and we don't see the page reload when clicking these links
```suggestion
        link_to path, remote: true, title: do
```